client/my-sites/media/main.jsx

```diff
@@ -15,7 +15,7 @@ import { localize } from 'i18n-calypso';
 import MediaLibrary from 'my-sites/media-library';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import Dialog from 'components/dialog';
-import { EditorMediaModalDetail } from 'post-editor/media-modal/detail';
+import EditorMediaModalDetail from 'post-editor/media-modal/detail';
```

Comment:
Can the `export const` be removed from this file as well?

Reply:
Hmm, good point. I was foolishly assuming it was being exported separately for testing, but that assumption doesn't seem to hold...